### ISSUE TITLE
pydantic v2 Warnings: The `dict` method is deprecated; use `model_dump` instead

### ISSUE THREAD

**Issue by bVdCreations** [melvinkcx/fastapi-events#48]
Hi, after migrating to pydantic v2 I have notice a lot of warnings popping up.

```bash
pydantic.warnings.PydanticDeprecatedSince20: The `dict` method is deprecated; use `model_dump` instead. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2
 Migration Guide at https://errors.pydantic.dev/2.1.1/migration/
 ```
 
 This is because of the dispatch `fastapi_events/dispatcher.py:119: in dispatch`
   `payload = payload_schema_cls(**(payload or {})).dict(**payload_schema_cls_dict_args)`

**Comment by melvinkcx** [comment]
Hi, thanks for reporting this, we'll have some time before this becomes an issue. I'll look into this perhaps in a week or so. Meanwhile, feel free to take a look at this if you're interested. :)

**Comment by bVdCreations** [comment]
@melvinkcx I will see if I find some time this weekend

**Comment by melvinkcx** [comment]
This has been resolved and released in v0.10.1
